tests/metrics/density/docker_memory_usage.sh.in

```diff
@@ -22,43 +22,89 @@
 #  This test launches certain number of containers and puts them into
 #  sleep mode for a certain period of time. Then the test checks the
 #  amount of memory used by each container
-#  This test uses psstop tool to get the memory used.
+#  This test uses smem tool to get the memory used.
 
 set -e
 
-[ $# -ne 2 ] && ( echo >&2 "Usage: $0 <times to run> <wait time>"; exit 1 )
+[ $# -ne 2 ] && ( echo >&2 "Usage: $0 <cc number> <wait time>"; exit 1 )
```

Comment:
I'd be tempted to create a `usage` function so you can add a bit more explanation here (such as the format of `<wait time>` and what `<cc number>` means. Something like:

```
function usage(){
  cat <<EOT
  Usage: $0 <count> <wait_time>

Where:

<count>        : number of Clear Containers to run.
<wait_time> : time in seconds to wait ... .

EOT
}